src/containers/TopBlocksContainer.jsx

```diff
@@ -1,8 +1,13 @@
 import {connect} from 'react-redux';
 import TopBlocks from '../components/TopBlocks.jsx';
+import {getRpcServer} from '../actions';
+
+const mapDispatchToProps = (dispatch) => ({
+  getRpcServer: () => dispatch(getRpcServer())
+});
 
 const mapStateToProps = (state) => ({
   blockList: state.topBlocks.blockList
 });
 
-export default connect(mapStateToProps)(TopBlocks);
+export default connect(mapStateToProps, mapDispatchToProps)(TopBlocks);
```

Comment:
Correct me if I'm wrong here @TalAter, but I believe the changes to this file are unnecessary. You are making a call to `getRpcServer()` inside of the `topBlocks` reducer.